### ISSUE TITLE
Survival Analysis 

### ISSUE THREAD

**Issue by gfrc11752** [Huh/PopR_SDGFP#109]
White-tailed Deer, DAU 9, 2016, Adult Female

In the fate column we have 6 individuals that died from capture myopathy. However, they are showing up as dead=true and censored= false. I am under the impression they should be dead= true and censored =true. Is that being derived from the input file or the code?

**Comment by Huh** [comment]
For your site if dead = T then censored has to be F.  We don't allow dead and censored to be T.  We can change this, but if you click exclude on the setup tab then those animals will not be in the analysis.

**Comment by gfrc11752** [comment]
Yeah, animals we censor should never be included in the survival analysis. If censor= TRUE then that individual should not be included in the analysis even if it is dead.

**Comment by Huh** [comment]
But that is very different than how we are using censor here, no?  If an animal was monitored for 6 months and then the collar failed it should be censored at the last time it was heard and included in the analysis.
I can make the default for exclude capture myopathy true so the box is checked.  
 
Josh Nowak 

    On Monday, April 4, 2016 11:29 AM, gfrc11752 <notifications@github.com> wrote:
 

 Yeah, animals we censor should never be included in the survival analysis. If censor= TRUE then that individual should not be included in the analysis even if it is dead.

**Comment by gfrc11752** [comment]
Yeah but in your example Dead = False

**Comment by Huh** [comment]
Yeah and consistent with the idea that an animal can't be censored and dead.  The best example I know of to have animals dead and censored is when a collar fails and the animal is harvested at some later date, but using the later date is not a great idea and likely to bias results.
Regardless the point I wanted to make with my example was that animals that are censored have to be included in the analysis.  Lots of animals will provide some information about being alive and then randomly leave the study for a number of reasons that are not death.  A collar reaching the end of the deployment and falling off is another example of that.  
The censor column helps us right censor animals and the exclude check boxes are there to exclude animals form the analysis completely.  I do think the labeling of capture myopathy as a type of censor is inconsistent and should probably be changed.  Josh Nowak 

    On Monday, April 4, 2016 11:50 AM, gfrc11752 <notifications@github.com> wrote:
 

 Yeah but in your example Dead = False

**Comment by gfrc11752** [comment]
I agree with all those things you mentioned; however, why wouldn’t we always censor all capture myopathies? If Capture myopathy, dead = true and censor = true. Am I missing something here, please straighten me out.

**Comment by Huh** [comment]
No, I think we are saying the same thing largely.  It is just the way it is implemented that is making me speak about it differently.  
I will make the default to remove all animals that have a censor type of capture myopathy.  Are there other sources of censoring or death that should always be removed? Josh Nowak 

    On Monday, April 4, 2016 12:10 PM, gfrc11752 <notifications@github.com> wrote:
 

 I agree with all those things you mentioned; however, why wouldn’t we always censor all capture myopathies? If Capture myopathy, dead = true and censor = true. Am I missing something here, please straighten me out.

**Comment by gfrc11752** [comment]
Alright, that sounds good. No, capture myopathy would be the only source of death that would always be censored. Collar tore off and lost signal would result in a death = false, censored  = true,  I believe.
